task/factory/README.md

```diff
@@ -0,0 +1,10 @@
+# Task Factory
+[Task](https://github.com/Knotx/knotx-fragments/blob/master/task/api#task) is an abstract structure:
+> Task is an identifiable graph that describes the way fragment should be processed.
+
+Task Factory materializes a task with using its logic:  
+> Task factory creates a ready to use task instance.
```

Comment:
I think this line just complicates things and does not give any value to the reader:
```suggestion
Task factory creates a ready to use task instance.
```